### DISCUSSION TITLE
An appeal to the developers regarding Frostivus 2023.

### DISCUSSION THREAD

**Discussion by 7ReaperOfSouls7** [ValveSoftware/Dota2-Gameplay#14615] [Ideas]
Comrade developers, you don’t feel the New Year’s mood and holiday because of paid chests with a key and unnecessary pictures, unnecessary emoticons and unnecessary graffiti.
Where is the promised content for all players?
At the moment, there is again only one donation content for paid players in chests and garbage in stockings.
In stockings, the only thing players need is a snowman ward, which is very difficult to get due to the fact that you need to play at least 200 games; the rest of the gifts in stockings are garbage.
Give your players a real New Year's holiday!
Please make it possible to open chests without keys and you need to update the content in stockings to normal content (appearances for heroes, couriers, new wards) instead of unnecessary garbage (pictures, emoticons, graffiti).
Hope for your understanding!

**Comment by nyczalex** [comment]
Yes, for a season of giving, this surely all that hyped. The stocking gifts are not very much sought after for the amount of commitment one needs to put to get them..   They really need start making it more rewarding for hardcore players that commit to unlocking them with cooler gifts.. 

As for the chests, it's been a problem, not only for strictly this chest but for any. The problem isn't that it costs keys to open but buying it off the market is **always** significantly cheaper. They need to really balance the gamble on these chests so people will actually buy it rather than visit the steam market.. If they did this, grinding players can sell the chests while others will buy them to gamble. 

Happy Holidays!
